src/components/Body/Portafolio/ItemPortafolio.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Ability from '../../../actions/Ability';
+
+
+class ItemPortafolio extends Component {
+
+  constructor(ability){
```

Comment:
oohh bad practice, you shouldn't pass params to the constructor, it just needs to receive the component props. This must be a stateless component. I have no idea how is this working 😅